node_archives.bzl

```diff
@@ -62,3 +72,13 @@ def repositories():
         architecture = "arm64",
         control = "//nodejs:control",
     )
+
+    node_archive(
+        name = "nodejs20_arm64",
+        sha256 = "997ddc7ca941280aeb357d4ad40ea7f93cb4d7989b93056dbf1b42a1a89b449b",
```

Comment:
hrmm these checksums appear to be wrong, I don't know why CI didn't pick this up?